.github/workflows/auto-add-parachain-issues.yml

```diff
@@ -0,0 +1,30 @@
+# If there are new issues related to the async backing feature,
+# add it to the parachain team's board and set a custom "meta" field.
+
+name: Add selected issues to Parachain team board
+on:
+  issues:
+    types:
+      - labeled
+
+jobs:
+  add-parachain-issues:
+    if: github.event.label.name == 'T16-async_backing'
+    runs-on: ubuntu-latest
+    steps:
+      - name: Generate token
+        id: generate_token
+        uses: tibdex/github-app-token@v2.1.0
+        with:
+          app_id: ${{ secrets.PROJECT_APP_ID }}
+          private_key: ${{ secrets.PROJECT_APP_KEY }}
```

Comment:
Would it be better to put this in an environment?

The same as [`review-bot`](https://github.com/paritytech/polkadot-sdk/blob/master/.github/workflows/review-bot.yml#L12), just to keep it extra safe.

Reply:
From the issue regarding app installation:

> This app is non-privileged, i.e. it doesn't have any sensitive access, therefore I installed the secrets outside of the environments, on the repo level.